package.json

```diff
@@ -7,6 +7,7 @@
         "@loadable/component": "^5.13.2",
         "commander": "^6.1.0",
         "crc-32": "^1.2.0",
+        "embla-carousel": "^4.0.5",
```

Comment:
can we remove react-id-swiper at once

Reply:
We are not using it anymore so yes it is possible